package.json

```diff
@@ -4,7 +4,7 @@
   "author": {
     "name": "Microsoft Corporation"
   },
-  "version": "0.10.0",
+  "version": "1.0.0",
```

Comment:
Could you please add a Git tag for this release, and maybe even Github Release as earlier releases have had?

Currently there is no information available about this major release. When automatic dependency updates are prompting this breaking change release for dependent projects, this v1 release doesn't look that reliable.

Reply:
I honestly thought I'd been typo squatted since npmjs.org has such newer versions than this github repo.